patches/minecraft/net/minecraft/block/SoundType.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/block/SoundType.java
++++ b/net/minecraft/block/SoundType.java
+@@ -3,6 +3,7 @@
+ import net.minecraft.util.SoundEvent;
+ import net.minecraft.util.SoundEvents;
+ 
++// Forge: Use {@link net.minecraftforge.common.util.ForgeSoundType} instead for suppliers
```

Comment:
I would move this comment to the Constructor, and then @Deprecate the constructor. As the class itself isn't deprecated, it's the new instances that are.

Reply:
Moved the comment, and deprecated the constructor for `SoundType`.